src/index.html

```diff
@@ -7,12 +7,71 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="container">
+      <section
+        class="container__section"
+        data-qa="card"
+      >
+        <img
+          class="product-image"
+          src="./images/imac.jpeg"
+          alt="iMac image"
+        />
+
+        <div class="container__description">
+          <h4 class="product-title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h4>
+          <p class="product-code">Product code: 195434</p>
+        </div>
+
+        <div class="review-section">
+          <div class="stars-counter">
+            <div class="star-active"></div>
+            <div class="star-active"></div>
+            <div class="star-active"></div>
+            <div class="star-active"></div>
+            <div class="star"></div>
+          </div>
```

Comment:
should be the separate stars block

Reply:
Don't understand what do you mean. Is that block in not separate?

Reply:
<img width="596" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/bb1f044d-cddc-4519-9298-49dec1c8c103">
in the task it is written that you should have two blocks "card" and "stars" (each of them in a separate file), not a review-section, price or container